tests/transactional/enable_selinux.pm

```diff
@@ -29,10 +39,26 @@ sub run {
         assert_script_run('transactional-update setup-selinux');
         upload_logs('/var/log/transactional-update.log');
         save_and_upload_log('rpm -qa', 'installed_pkgs.txt');
-        process_reboot(trigger => 1);
+        reboot;
+    }
+
+    # On PublicCloud we need to enable SELinux via kernel boot parameter
```

Comment:
but this is what `transactional-update setup-selinux` does for you, or isn't not the case for PC?

Reply:
Nope.